crates/store/re_types_core/src/component_descriptor.rs

```diff
@@ -85,6 +85,22 @@ impl ComponentDescriptor {
         self.sanity_check();
         self.component.as_str()
     }
+
+    /// Formats the descriptor with archetype and display name.
+    pub fn format_descriptor(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.write_str(":")?;
+        f.write_str(self.display_name())?;
+        if let Some(archetype) = self.archetype {
+            f.write_str("@")?;
+            f.write_str(&archetype)?;
+        }
+        if let Some(ty) = self.component_type {
+            f.write_str("#")?;
+            f.write_str(&ty)?;
+        }
+
+        Ok(())
+    }
```

Comment:
do we still need this now? I'm actually not sure about the exact convention we should go for here since rerun-component-identifiers also embed the archetype name.
Plus, starting with a `:` is strange